Readme (#56)
web3j/web3j-openapi: README.md

- Sending HTTP requests using Curls or Swagger-UI

### Why an OpenAPI generator
As stated above, the Web3j-OpenAPI generator generates an OpenAPI project from a smart contract. Thus, providing a way to interact with the Etheruem blockchain using HTTP requests.

Such generator is beneficial in the following ways:
#### Making it easier to interact with the Ethereum blockchain:
Interacting with the Ethereum blockchain before required knowing a programming language besides solidity, and then hardcoding the desired logic. This, makes it hard for people wishing to get involved in the smart contracts world and adds extra technical requirements in addition to knowing the smart contracts language.
#### Interact with smart contracts without code:
Being able to generate an OpenAPI project from a smart contract and interacting with it using HTTP requests, eliminates the need to code any interactions and being able to send them to the blockchain. Thus, it is legitimate to say that the Web3j-OpenAPI lets you interact with the Ethereum blockchain using no code, besides solidity.

## Use cases
An OpenAPI project can be generated using the following command:

```ssh

> $ epirus generate openapi -p com.helloworld -o . --abi helloworld.abi --bin helloworld.bin --name helloworld

Reviewer: this part of the documentation needs to go into epirus-cli. 

you can link to epirus-cli here instead

Author: what about now ?

```

Then, the generated project can be used in the following ways:

**Creating a Gradle project that can be run using the application plugin:**
```ssh
$ cd helloworld
$ ./gradlew run // Starts the server exposing Helloworld.sol
...
```

**Starting the server using the ShadowJar:**
```ssh
$ ./gradlew shadowJar // Create an executable JAR that starts the server
$ java -jar helloworld-server-all.jar
...
```

> **Using the CLI:**

Reviewer: ```suggestion
**Starting the server using the CLI:**
```

[![Build Status](https://travis-ci.org/web3j/web3j-openapi.svg?branch=master)](https://travis-ci.org/web3j/web3j-openapi)

Web3j-OpenAPI is an OpenAPI generator from solidity smart contracts. 

## Description 
Web3j-OpenAPI provides a way to interact with the Ethereum blockchain via simple and intuitive HTTP requests, abstracting the coding layer. These interactions can be done using plain HTTP requests or via the Swagger-UI, which is generated with every project.

The workflow can be summed in the following steps:
- Writing a solidity smart contract

> - Generating the corresponding OpenAPI project using Web3j-OpenAPI

Author: ```suggestion
- Generating the corresponding **OpenAPI** project using **Web3j-OpenAPI**
```

- Running the generated project
- Sending HTTP requests using Curls or Swagger-UI

> **For more explanations**, check the documentation, this blog post, or the demo project.

Author: ```suggestion
**For more explanations**, a well detailed documentation and a sample project are on the way.
```
